src/app/pages/datasets/components/dataset-form/utils/dataset-form.service.spec.ts

```diff
@@ -37,7 +40,7 @@ describe('DatasetFormService', () => {
         helptextDatasetForm.pathWarningTitle,
         helptextDatasetForm.pathIsTooLongWarning,
       );
-      expect(spectator.inject(OldSlideInService).closeLast).toHaveBeenCalled();
+      expect(spectator.inject(SlideIn).popComponent).toHaveBeenCalled();
```

Comment:
`closeLast` makes it easy to understand what's going on.
`popComponent(undefined)` - not so much.